core/store/src/db.rs

```diff
@@ -145,12 +145,12 @@ fn col_name(col: DBCol) -> String {
 fn ensure_max_open_files_limit(max_open_files: i32) -> () {
     // We’re configuring each RocksDB to use max_open_files file descriptors. On top of that we can
     // have some other file descriptors opened by neard process so we use the value of
-    // 2 * max_open_files to be sure that the binary can correctly run.
+    // max_open_files + 1025 to be sure that the binary can correctly run.
     let (soft, hard) = rlimit::Resource::NOFILE.get().unwrap();
     let required = max_open_files as u64 + 1025;
     if soft < required {
-        assert!(hard >= required, concat!("Can't run near binary since hard limit for the number ",
-                "of opened files is too small: {} required: {}"), hard, 2 * max_open_files);
+        assert!(hard >= required, "Can't run near binary since hard limit for the number \
+                of opened files is too small: {} required: {}", hard, 2 * max_open_files);
```

Comment:
This still needs to be `required` in the last argument.

Reply:
sorry for that I thought you were talking about the boolean expression